### PR TITLE
Update the netops README.md file

### PR DESCRIPTION
**Pull request by uma-kt**
- update the hyperlinks
- Rephrased the sentences.
- Add image to the docke hub image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 The netop pods enforce the QoS Profile for a slice. It uses Linux TC (Traffic Control) for the slice traffic classification.
```
Comment (veenamj-avesha) [inline README.md:4]: NetOps

File: README.md [hunk 2]
```diff
@@ -18,15 +18,15 @@ Before you begin, make sure the following prerequisites are met:
 * You have prepared the environment to install [`kubeslice-controller`](https://github.com/kubeslice/kubeslice-controller) on the controller cluster
  and [`worker-operator`](https://github.com/kubeslice/worker-operator) on the worker cluster. For more information, see [Prerequisites](https://kubeslice.io/documentation/open-source/0.5.0/getting-started-with-cloud-clusters/prerequisites/).
  
-### Local Build and Update 
+### Build and Deploy netops on a Kind Cluster
```
Comment (veenamj-avesha) [inline README.md:21]: NetOps

File: README.md [hunk 3]
```diff
@@ -18,15 +18,15 @@ Before you begin, make sure the following prerequisites are met:
...
 
-#### Latest Docker Hub Image
+To download the latest netops docker hub image, click [here](https://hub.docker.com/r/aveshasystems/netops).
```
Comment (veenamj-avesha) [inline README.md:23]: NetOps

File: README.md [hunk 4]
```diff
@@ -49,44 +49,44 @@ git clone https://github.com/kubeslice/netops.git
...
-Image is set as `docker.io/aveshasystems/netops:$(VERSION)` in the Makefile. Change this if required.
+2. Edit the `VERSION` variable in the Makefile to change the docker tag to be built.
+Image is set as `docker.io/aveshasystems/netops:$(VERSION)` in the Makefile. Modiy this if required.
 
 ```bash
 make docker-build
 ```
 
-### Running Locally on Kind Clusters
+### Run Locally on a Kind Clusters
```
Comment (veenamj-avesha) [inline README.md:59]: `on a Kind Cluster` or `on Kind Clusters`

File: README.md [hunk 5]
```diff
@@ -49,44 +49,44 @@ git clone https://github.com/kubeslice/netops.git
...
+   docker exec -it kind-control-plane crictl images
+   ```
 
 
-### Deploy in a Cluster
+### Deploy on a Cluster
 
-Update the chart values file called `yourvaluesfile.yaml` that you have previously created.
-Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-worker/values.yaml) to create `yourvaluesfiel.yaml` and update the netop image subsection to use the local image.
+Update a chart values file called `yourvaluesfile.yaml` that you have previously created.
```
Comment (veenamj-avesha) [inline README.md:88]: `the` is correct  as the file with the same name existed and we are just asking to update that file.

File: README.md [hunk 6]
```diff
@@ -49,44 +49,44 @@ git clone https://github.com/kubeslice/netops.git
...
+Refer to the [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-worker/values.yaml) to create `yourvaluesfiel.yaml` and update the netop image subsection to use the local image.
```
Comment (veenamj-avesha) [inline README.md:89]: NetOps

**Review by veenamj-avesha — COMMENTED** [review]
I've left a few comments.

File: README.md [hunk 7]
```diff
@@ -49,44 +49,44 @@ git clone https://github.com/kubeslice/netops.git
 cd netops
 ```
 
-2. Adjust the `VERSION` variable in the Makefile to change the docker tag to be built.
-Image is set as `docker.io/aveshasystems/netops:$(VERSION)` in the Makefile. Change this if required.
+2. Edit the `VERSION` variable in the Makefile to change the docker tag to be built.
+Image is set as `docker.io/aveshasystems/netops:$(VERSION)` in the Makefile. Modiy this if required.
```
Comment (veenamj-avesha) [inline README.md:53]: The image is set ...

File: README.md (hunk 3 above)
Comment (veenamj-avesha) [reply README.md:23]: comment is not fixed

**Review by veenamj-avesha — COMMENTED** [review]
A previous comment is not fixed.

File: README.md (hunk 3 above)
Comment (uma-kt) [reply README.md]: updated

**Review by veenamj-avesha — APPROVED** [review]
Changes LGTM

**Pull request merged**